pkg/users/logout-account-dialog.js

```diff
@@ -44,10 +42,5 @@ export const logoutAccountDialog = (account) => {
         ]
     };
 
-    if (!dlg)
-        dlg = show_modal_dialog(props, footer);
-    else {
-        dlg.setProps(props);
-        dlg.setFooterProps(footer);
-    }
+    show_modal_dialog(props, footer);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18008-20221208-172856-e7737fdb-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)